packages/sku/plop/helpers/camel-case-helper.mjs

```diff
@@ -0,0 +1,9 @@
+import camelCase from 'lodash/camelCase.js';
```

Comment:
I've developed an allergy towards lodash lately. It's not inherently bad, but I'd rather not add it as a dependency (even as a dev dep) if we're just using a single utility. Since this is ESM, we could use [`camelcase`](https://www.npmjs.com/package/camelcase) instead.